### ISSUE TITLE
[BUG] Postgres Data Exporter ignores case

### ISSUE THREAD

**Issue by webserviceXXL** [mage-ai/mage-ai#4740]
### Mage version

0.9.65

### Describe the bug

If the Dataframe contains columns with camelCase names they are transformet to lowercase when using the Postgres export.

```
    with Postgres.with_config(ConfigFileLoader(config_path, config_profile)) as loader:
        loader.export(
            df,
            schema_name,
            table_name,
            index=False,  # Specifies whether to include index in exported table
            if_exists='append',  # Specify resolution policy if table name already exists
        )
```

Error message: `UndefinedColumn: column "networkid" of relation "programs" does not exist`

### To reproduce

Create a Data Frame wir camelCase columns

### Expected behavior

Respect case

### Screenshots

![Screenshot-program import  Mage-Arc-20240312_1420@2x](https://github.com/mage-ai/mage-ai/assets/3533300/1d563440-6611-494f-835f-0b456f330a7d)
![Screenshot-program import  Mage-Arc-20240312_1420@2x](https://github.com/mage-ai/mage-ai/assets/3533300/90d7e70c-27a9-4d80-96a2-1fd0baf0dfa7)

**Comment by webserviceXXL** [comment]
I guess this happens because the query don't use quotes for the columns.

`SELECT networkId FROM table;` won't work in psql. For camlCase columns you have to use quotes like `SELECT "networkId" FROM table;`

**Comment by wangxiaoyou1993** [comment]
This is resolved in this PR: https://github.com/mage-ai/mage-ai/pull/4698 and published in version 0.9.66
You can set `auto_clean_name=False` param in the export method.

**Comment by webserviceXXL** [comment]
@wangxiaoyou1993 Awesone, thank you!
